torch/nn/quantized/modules/linear.py

```diff
@@ -8,19 +8,36 @@
 from torch.nn.quantized.modules.utils import _quantize_weight
 
 class LinearPackedParams(torch.nn.Module):
-    def __init__(self):
+    _version = 2
```

Comment:
Not sure if _version is needed as we do not need to have dtype be part of the state dict.

Reply:
See below, `dtype` is definitely needed